validator/testdata/feature_tests/root_element_attributes.out

```diff
@@ -16,7 +16,9 @@ FAIL
 |    This demonstrates that certain attributes are not allowed on the root <html> element.
 |  -->
 |  <!doctype html>
-|  <html ⚡ allow-xhr-interception>
+|  <html ⚡ allow-xhr-interception allow-error-reporting-to-viewer>
```

Comment:
`report-errors-to-viewer` here and below.

Reply:
Done.